app/main.py

```diff
@@ -1,8 +1,44 @@
+from typing import Any
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: int) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int, clean_power: int,
+                 average_rating: int, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, list_car: list) -> float:
+        profit = 0
+        for car in list_car:
+            if car.clean_mark < self.clean_power:
+                profit += CarWashStation.calculate_washing_price(self, car)
```

Comment:
Why do you use CarWashStation? You should use `self`. Fix everywhere
```suggestion
                profit += self.calculate_washing_price(self, car)
```

Reply:
if I change to self then it doesn't see the argument car